### PR TITLE
Payment flow - All redux functions created

### PR DESCRIPTION
**Pull request by prateekchachra**

### CODE REVIEW

File: src/pages/Payment/Payment.tsx [hunk 1]
```diff
@@ -9,27 +9,87 @@ import PriceSummary from '../../components/organisms/PriceSummary/PriceSummary';
...
+      });
+
+    const [paymentOption, setPaymentOption] = useState<string>('');
     const [showConfirmModal, setShowConfirmModal] = useState<boolean>(false);
-    const [refundId, setRefundId] = useState('');
+    const [refundId, setRefundId] = useState<string>('');
+
+    //For the card form
+
+    const [cardNumber, setCardNumber] = useState<string>('')
```
Comment (DParik) [inline src/pages/Payment/Payment.tsx:60]: Should this be a string or a number type?

**Review by DParik — COMMENTED** [review]
I am not able to open up the second login modal for google login. Maybe we can try to merge this PR with main and test over there only.

**Review by DParik — COMMENTED** [review]
May I ask - Where can I find the logic which stores cartState and wishlistState into DB? (using axios)

File: src/pages/Payment/Payment.tsx (hunk 1 above)
Comment (prateekchachra) [reply src/pages/Payment/Payment.tsx:60]: A 16 digit credit card number never has a number type :P If you want you can add validation for the same. Most welcome to work on it

**Pull request merged**